packages/language-server/src/plugins/svelte/SveltePlugin.ts

```diff
@@ -25,6 +25,29 @@ import { getHoverInfo } from './features/getHoverInfo';
 import { SvelteDocument } from './SvelteDocument';
 import { Logger } from '../../logger';
 
+export type SvelteCompiledResult = {
```

Comment:
What do you think about just use
```ts
import type { compile } from 'svelte/compiler';

export type SvelteCompiledResult = ReturnType<typeof compile>
```

Reply:
Good point. I thought that would also import `compile` function as value, that's why I manually wrote it. Turns out TS is smart enough to figure out that value can be dropped.

Reply:
Do you think it's better to use type only import?
```ts
import type { compile } from 'svelte/compiler';
```
It requires TS v3.8+ tho.

Reply:
We are using ts 3.9 now. but I think using normal import could probably be the same. It could be removed.